churchland_pipeline/tasks/pacman.py

```diff
@@ -22,10 +22,25 @@ class ArmPosture(dj.Lookup):
 class SimulinkState(dj.Lookup):
     definition = """
     # Simulink Stateflow state IDs and names
-    state_id : tinyint unsigned # unique task state ID number
+    task_state_id : tinyint unsigned # task state ID number
```

Comment:
It usually a good practice to keep the name of the table and the name of its identifying attributes similar..  For example, the name of the table could be `TaskState`.  The comment identifies that it is managed by Simulink.